### PR TITLE
Add optional status codes to exceptions

### PR DESCRIPTION
**Pull request by grzegorzpro**

### CODE REVIEW

File: lbz/exceptions.py [hunk 1]
```diff
@@ -15,20 +16,23 @@ class LambdaFWException(Exception):
...
-    def __init__(self, message: str = "") -> None:
+    def __init__(self, message: str = "", error_code: Optional[str] = None) -> None:
         super().__init__(message)
         if message:
             self.message = message
+        self.error_code = error_code
 
     def __str__(self) -> str:
-        return f"[{self.status_code}] {self.message}"
+        full_msg = f"{self.error_code} - {self.message}" if self.error_code else self.message
```
Comment (redlickigrzegorz) [inline lbz/exceptions.py:26]: Personally, I would avoid making double f-string, this is small enough to do something like this:
```python
if self.error_code is not None:
    return f"[{self.status_code}] {self.error_code} - {self.message}"
return f"[{self.status_code}] {self.message}"
```

File: tests/test_exceptions.py [hunk 2]
```diff
@@ -50,9 +50,26 @@
 
 def test_lambda_fw_exception() -> None:
     exp = LambdaFWException(message="Nope")
+    assert str(exp) == "[500] Nope"
     assert exp.message == "Nope"
     assert exp.status_code == 500
-    assert isinstance(exp.get_response(request_id=""), Response)
+
+    response = exp.get_response(request_id="test-req-id")
+    assert isinstance(response, Response)
```
Comment (redlickigrzegorz) [inline tests/test_exceptions.py:58]: I don't think this assertion is needed - with `mypy` and the below line, it will be pretty hard to make it wrong 😉

**Comment by pdyba** [comment]
OK after @redlickigrzegorz comments will be resolved

File: CHANGELOG.md [hunk 3]
```diff
@@ -224,3 +224,9 @@ Released 2022-06-04
 
 - Fixes EventAPI to send events in chunks (conforms to AWS limits)
 - Adds real property methods that protect events list the better way
+
+### Version 0.5.6
+Released 2022-07-25
+
+- Adds a decorator that makes the given function an event emitter
```
[the remark this answers is not in the document]
Comment (grzegorzpro) [reply CHANGELOG.md:231]: No problem! I could swear 0.5.6 was not published yet when I wrote this :o

[the remark this answers is not in the document]
Comment (redlickigrzegorz) [reply CHANGELOG.md:231]: There were some problems in PyPi yesterday, so depending on where you checked it, you could see different versions 🤷 

Rel. https://github.com/pypi/warehouse/issues/11939

**Review by redlickigrzegorz — APPROVED** [review]

**Pull request merged**